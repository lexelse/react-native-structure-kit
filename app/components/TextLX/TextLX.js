// @flow

import * as React from "react";
import { Text } from "react-native";

type Props = {
    children?: React.Node
}

const TextLX = (props: Props) => {
    const { children } = props;

    return (
        <Text>{children}</Text>
    );
};

export default TextLX;
