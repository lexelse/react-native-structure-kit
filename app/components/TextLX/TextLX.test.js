import React from "react";
import { shallow } from "enzyme";
import TextLX from "./TextLX";

describe("TextLX", () => {
    describe("Rendering", () => {
        it("should match to snapshot", () => {
            const component = shallow((<TextLX />)).debug();
            expect(component).toMatchSnapshot();
        });

        it("should match to snapshot with props.children", () => {
            const component = shallow((<TextLX>HELLO WORLD</TextLX>)).debug();
            expect(component).toMatchSnapshot();
        });
    });
});
