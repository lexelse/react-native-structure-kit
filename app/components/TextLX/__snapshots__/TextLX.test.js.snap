// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TextLX Rendering should match to snapshot 1`] = `ShallowWrapper {}`;
